templates/dashboard/support.html

```diff
@@ -0,0 +1,101 @@
+{% extends 'default.html' %}
+
+{% set active_page = None %}
+
+{% block title %}
+    Support
+{% endblock %}
+
+{% block head %}
+    <style>
+        .card-title {
+            font-size: 22px;
+            font-weight: 600;
+            margin-bottom: 3px;
+        }
+    </style>
+
+    <script src="{{ url_for('static', filename='node_modules/vue/dist/vue.min.js') }}"></script>
```

Comment:
Usually js part is included in the `{% block script %}`. This way we don't have to use `$( document ).ready(function()`. An example of vuejs usage is https://github.com/simple-login/app/blob/1fd9a344d4d728eb5da3e232b64881be5279ca61/static/js/index.js#L256